api/__init__.py

```diff
@@ -128,5 +130,7 @@ def add_headers(response):
 
     app.register_blueprint(messages_views.bp)
     app.register_blueprint(exception_views.bp)
+    app.register_blueprint(newsletter_views.bp)
+    app.register_blueprint(subscription_views.bp)
```

Comment:
Solid de-coupling from existing codebase @leonkoech - this is a great catch/knowledge to make this decision